v3.0/services.html

```diff
@@ -0,0 +1,139 @@
+			</ul>
+			<hr>
+			Memory Map
+			<ul>
+				<li><a href="fam_map.html">fam_map</a></li>
+				<li><a href="fam_unmap.html">fam_unmap</a></li>
+			</ul>
+			<hr>
+			Data Path Operations
+			<ul>
+				<li><a href="fam_get.html">fam_get</a></li>
+				<li><a href="fam_put.html">fam_put</a></li>
+				<li><a href="fam_gather.html">fam_gather</a></li>
+				<li><a href="fam_scatter.html">fam_scatter</a></li>
+				<li><a href="fam_copy.html">fam_copy</a></li>
+				<li><a href="fam_copy_wait.html">fam_copy_wait</a></li>
+				<li><a href="fam_backup.html">fam_backup</a></li>
+				<li><a href="fam_backup_wait.html">fam_backup_wait</a></li>
+				<li><a href="fam_restore.html">fam_restore</a></li>
+				<li><a href="fam_restore_wait.html">fam_restore_wait</a></li>
+				<li><a href="fam_delete_backup.html">fam_delete_backup</a></li>
+				<li><a href="fam_delete_backup_wait.html">fam_delete_backup_wait</a></li>
+
+			</ul>
+			<hr>
+			Atomics
+			<ul>
+				<li><a href="fam_set.html">fam_set</a></li>
+				<li><a href="fam_add.html">fam_add</a></li>
+				<li><a href="fam_subtract.html">fam_subtract</a></li>
+				<li><a href="fam_min.html">fam_min</a></li>
+				<li><a href="fam_max.html">fam_max</a></li>
+				<li><a href="fam_and.html">fam_and</a></li>
+				<li><a href="fam_or.html">fam_or</a></li>
+				<li><a href="fam_xor.html">fam_xor</a></li>
+				<li><a href="fam_fetch_TYPE.html">fam_fetch_TYPE</a></li>
+				<li><a href="fam_swap.html">fam_swap</a></li>
+				<li><a href="fam_compare_swap.html">fam_compare_swap</a></li>
+				<li><a href="fam_fetch_add.html">fam_fetch_add</a></li>
+				<li><a href="fam_fetch_subtract.html">fam_fetch_subtract</a></li>
+				<li><a href="fam_fetch_min.html">fam_fetch_min</a></li>
+				<li><a href="fam_fetch_and.html">fam_fetch_and</a></li>
+				<li><a href="fam_fetch_max.html">fam_fetch_max</a></li>
+				<li><a href="fam_fetch_or.html">fam_fetch_or</a></li>
+				<li><a href="fam_fetch_xor.html">fam_fetch_xor</a></li>
+			</ul>
+			<hr>
+			Ordering
+			<ul>
+				<li><a href="fam_barrier_all.html">fam_barrier_all</a></li>
+				<li><a href="fam_fence.html">fam_fence</a></li>
+				<li><a href="fam_quiet.html">fam_quiet</a></li>
+				<li><a href="fam_progress.html">fam_progress</a></li>
+				<li><a href="fam_context_open.html">fam_context_open</a></li>
+				<li><a href="fam_context_close.html">fam_context_close</a></li>
+			</ul>
+			<hr>
+		</nav>
+		<article>
+		<h1>OpenFAM 3.0 Services</h1>
+<p> OpenFAM version 2.0 introduced the concept of supporitng large pool of memory using multiple memory servers.
+OpenFAM version 3.0 contains significant design changes to increase scalability and improve performance.
+<ul>
+<li>The design supports a larger pool of memory by supporting multiple memory servers. This memory pool can be either volatile or persistent.</li>
+<li>Data can be interleaved as dataitems can now span across multiple memory servers, thus reducing performance bottlenecks.</li>
```

Comment:
Search for "dataitem"/"dataitems" throughout the documentation and replace with "data item"/"data items" (note the space between "data" and "item")